collections/datasets/emlhandler.php

```diff
@@ -3,6 +3,7 @@
 include_once($SERVER_ROOT.'/classes/DwcArchiverCore.php');
 
 $collid = isset($_REQUEST['collid']) ? filter_var($_REQUEST['collid'], FILTER_SANITIZE_NUMBER_INT) : '';
+if(!is_numeric($collid))$collid=0;
```

Comment:
Can't this just be combined with the above line? Isset followed by && is_numeric? filter stuff: 0;

Reply:
Fixed in 7c4162824. I think you're right that if it's not numeric right off the bat, we probably don't want it even after `FILTER_SANITIZE_NUMBER_INT`?